tests/network/expose.go

```diff
@@ -321,7 +317,7 @@ var _ = SIGDescribe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			},
 				Entry("over SingleStack IP family policy", k8sv1.IPFamilyPolicySingleStack),
 				Entry("over PreferDualStack IP family policy", k8sv1.IPFamilyPolicyPreferDualStack),
-				Entry("over RequireDualStack IP family policy", k8sv1.IPFamilyPolicyRequireDualStack),
+				Entry("over RequireDualStack IP family policy", decorators.RequiresDualStackCluster, k8sv1.IPFamilyPolicyRequireDualStack),
```

Comment:
What about the `Entry("over PreferDualStack IP family policy", k8sv1.IPFamilyPolicyPreferDualStack),`?

Reply:
it is checked already line above,
here we "enforce" dual

Reply:
> What about the `Entry("over PreferDualStack IP family policy", k8sv1.IPFamilyPolicyPreferDualStack),`?

If I’m not mistaken, that one does not “require” dualstack, it only “prefers” it.

Reply:
I am curios, how do you check the preference if you can't?

Reply:
Maybe I’m missing something, but in both cases it is checked, one when there is and the other when there is not.

The existing skip also treated the required scenario, not the preferred one.

Reply:
Now we would skip required only if the lanes aren't dual stack
all our vm based lanes beside one is dual stack
and we also have sanity check for that on kubevirtci check-up iirc